.travis.yml

```diff
@@ -14,7 +14,7 @@ addons:
       - g++-4.8
 
 before_script:
-  - npm install -g gulp
-  - for d in `ls packages/`; do cd packages/$d; npm install; cd -; done
+  - npm install -g gulp lerna
+  - lerna bootstrap
```

Comment:
Yeah, so much cleaner!

Incidentally I couldn't get lerna to link the two packages together without requiring `react-server-gulp-module-tagger` with a caret.  It [_looks_](https://github.com/kittens/lerna/blob/master/lib/commands/bootstrap/linkDependenciesForPackage.js#L48) like an exact version match should work.  I haven't traced it through to figure out why it didn't.